bin/plugin/commands/performance.js

```diff
@@ -118,11 +120,18 @@ function formatTime( number ) {
 /**
  * Curate the raw performance results.
  *
+ * @param {string} testSuite
  * @param {WPRawPerformanceResults} results
  *
  * @return {WPPerformanceResults} Curated Performance results.
  */
-function curateResults( results ) {
+function curateResults( testSuite, results ) {
+	if ( testSuite === 'front-end' ) {
```

Comment:
The `WPPerformanceResults` and `WPRawPerformanceResults` data structures expect the results to be the same shape. The front-end metrics will be different from the editor ones, so this will no longer be true. I'm not very keen of stretching it this way, though it's simple and works.

An alternative I considered was to make the tests themselves return a single value. The test would be responsible for curating the value it wants to report, not the performance command. I'm tempted to do it. The only benefit of having an array of values here would be to aggregate them all across the different runs, which is something we don't do at the moment anyway. FWIW, we only do 1 run at the moment (though we did 3 in the past).

Reply:
Another advantage of asking the test to curate the results themselves is that adding a new metric doesn't involve modifying the performance command, only the test itself.